src/components/Icons/Icon.stories.mdx

```diff
@@ -186,6 +186,13 @@ export const allIconsType = (list) => (
   <ul style={{ listStyle: "none" }}>{list}</ul>
 );
 
+## Accessibility
```

Comment:
Can you also add something to the `title` prop's description stressing the importance of the prop when it comes to accessibilty?